TranslatorReasonerAPI.yaml

```diff
@@ -632,15 +632,27 @@ components:
             Corresponds to the map key identifier of the
             object concept node anchoring the query filter
             pattern for the query relationship edge.
-        constraints:
+        attribute_constraints:
           type: array
           description: >-
-            A list of contraints applied to a query edge.
+            A list of attribute contraints applied to a query edge.
             If there are multiple items, they must all be true (equivalent
             to AND)
           items:
             $ref: '#/components/schemas/QueryConstraint'
           default: []
+        qualifier_constraints:
+          type: array
+          description: >-
+            A list of QualifierSets that provide nuance to the QEdge.
+            If multiple QualifierSets are provided, there is an OR
+            relationship between them. If the QEdge has multiple
+            predicates, then each QualifierSet SHOULD define the
+            predicate it is qualifying via the
+            QualiferSet.qualified_predicate property.
+          items:
+            $ref: '#/components/schemas/QualifierSet'
```

Comment:
As noted above, consider renaming 'QualifierSet' --> 'QualifierConstraint' or 'QualifierQueryConstraint', to be more clear and consistent with the name of the object that holds a constraint definition based on Attribute values. 

Also, this avoids issue that a 'QualifierSet' object as presently defined doesn’t actually hold a set of Qualifiers.  It holds one Qualifier (which is comprised of a key-value pair of fields), along with optional predicate it is linked to.

Reply:
After pondering further, I'm convinced that QualifierConstraint is better. I have updated the PR.